website/templates/game/shares.html

```diff
@@ -0,0 +1,46 @@
+{% extends "template.html" %}
+
+{% block title %}Parts{% endblock %}
+{% block h1 %}Parts{% endblock %}
+
+{% block content %}
+{% include "_pager.html" with base_url="website.views.data.shares" base_title="Consulter les parts au tour" %}
+
+<table class="prop_table slanted_head">
+<thead>
+<tr>
+	<th></th>
+	{% for total in totals %}
+	<!-- should be replaced with the player's image when -->
+	<th><div><span>{{ total.player.name|title }}</span></div></th>
```

Comment:
`<th>{{ total.player.name|title }}</th>`

(everything else was for slanted CSS)